src/transformers/tokenization_utils_base.py

```diff
@@ -1192,7 +1192,11 @@ def special_tokens_map(self) -> Dict[str, Union[str, List[str]]]:
         for attr in self.SPECIAL_TOKENS_ATTRIBUTES:
             attr_value = getattr(self, "_" + attr)
             if attr_value:
-                set_attr[attr] = str(attr_value)
+                set_attr[attr] = (
+                    [str(attr_value_sub) for attr_value_sub in attr_value]
```

Comment:
Since we test for several types afterward, I would preserve the type:
```suggestion
                    type(attr_value)(str(attr_value_sub) for attr_value_sub in attr_value)
```

Reply:
Good idea!